### ISSUE TITLE
Não encontro a extensão em meu vs code!!!

### ISSUE THREAD

**Issue by NathanGowacki** [Sneezry/vscode-watch-in-chrome#5]
Não consigo encontrar a extensão watch in Chrome em meu VS Code para baixar, não aparece na pesquisa

**Comment by MarceloBaliza** [comment]
Tenta pelo nome do fornecedor: SNEEZRY

**Comment by Thaina85** [comment]
parece que parou de funcionar mesmo.

**Comment by Thaina85** [comment]
Estou com dificuldades para fazer uma section (com duas divs) no VSC. O Watch in Chrome nao está funcionando mais e nem consigo achar ele mais no meu editor. Assim que eu clico em ctrl+shift+p sou direcionada para o Internet explorer. Nao sei se é impressao minha mas nao é a mesma coisa. Estou preicisando resolver um exercício aqui e nao consigo porque na parte do css simplesmente nao aparece a parte que eu programei. Já revi os meus códigos várias vezes e nao vejo nada de errado.

**Comment by wsagv** [comment]
Também não consigo encontrar a watch in Chrome em meu VS Code.

**Comment by ThainaSchumann** [comment]
Parece que foi descontinuada. Eu instalei o Live server e ta tranquilo!

**Comment by pedrogxv** [comment]
Aparentemente ela saiu do Marketplace.

**Comment by SrTrindade** [comment]
A Extensão foi desabiltiada pelo Desenvolvedor. Pra quem estiver procurando uma parecida tem a "Live Server". Espero que algum dia essa volte também.

**Comment by ghost** [comment]
> A Extensão foi desabiltiada pelo Desenvolvedor. Pra quem está procurando uma parecida tem um "Live Server". Espero que algum dia essa volte também.

Obrigado pela dica, amigo.

**Comment by devlobao84** [comment]
Uma dúvida, estou ainda aprendendo na área de Dev, inclusive sou novo no VS e até aqui no Github. No meu caso, quando estava fazendo o curso da Origamid, lembro que instalei o Live Serve, então como o o Watch in Chrome pelos que vcs estão dizendo foi descontinuado, o LIVE SERTVE dá pra usar de boa?: Seria a mesma coisa, ou quase??

**Comment by ThainaSchumann** [comment]
Sim! devlobao84. Eu uso o Live Serve e na minha opinião é tão bom quanto o Watch in Chrome. Eu tmb estou iniciando na área de Dev e estou aprendendo muito. Se alguém tiver um outro palpite, vou adorar saber. Espero ter ajudado. Boa sorte!

**Comment by devlobao84** [comment]
Opa , valeu @Tha7na  pela dica, ajudou muito sim, to aprendendo ainda, sou mais da área de design, mas to entrando na área de front end e UI Design também... Sucesso sempre!!

**Comment by Lucas-Machado-Alves** [comment]
Obrigado pessoal, os comentários de vocês me ajudaram bastante, valeu!!!

**Comment by Carloschaveiro** [comment]
Estou usando o Studio Visual Codes , a procura do plug-in Watch in Chrome , fui ao GitHub no fórum  e nos comentários disseram q o LIVE SERTVE , fui instalei e ficou perfeito. Muito obrigado.

**Comment by antoniosud** [comment]
> Parece que foi descontinuada. Eu instalei o Live server e ta tranquilo!

Valew!! Ajudou demais.

**Comment by vv1101** [comment]
pra mim nem live server ta funcionando

**Comment by estefaneSantos** [comment]
Muito obrigada, vcs ajudaram demais, iniciei o curso de JS e o prof pedia pra instalar o Watch in Chrome e eu ja tava desesperada por que não estava aparecendo no meu, vou ver se o Live server vai da certo. espero que sim.

**Comment by ingridcso** [comment]
> Muito obrigada, vcs ajudaram demais, iniciei o curso de JS e o prof pedia pra instalar o Watch in Chrome e eu ja tava desesperada por que não estava aparecendo no meu, vou ver se o Live server vai da certo. espero que sim.

professor guanabara? se for, estamos juntas o/

**Comment by antoniosud** [comment]
Guanabara sim.
________________________________
De: ingridcso ***@***.***>
Enviado: segunda-feira, 24 de maio de 2021 20:23
Para: Sneezry/vscode-watch-in-chrome ***@***.***>
Cc: Antonio Costa ***@***.***>; Comment ***@***.***>
Assunto: Re: [Sneezry/vscode-watch-in-chrome] Não encontro a extensão em meu vs code!!! (#5)


Muito obrigada, vcs ajudaram demais, iniciei o curso de JS e o prof pedia pra instalar o Watch in Chrome e eu ja tava desesperada por que não estava aparecendo no meu, vou ver se o Live server vai da certo. espero que sim.

professor guanabara? se for, estamos juntas o/

**Comment by IT-Leandro-Camargo** [comment]
o meu Live Server não roda a cada alteração que faço no VS

**Comment by MonicaDalosto** [comment]
> > Muito obrigada, vcs ajudaram demais, iniciei o curso de JS e o prof pedia pra instalar o Watch in Chrome e eu ja tava desesperada por que não estava aparecendo no meu, vou ver se o Live server vai da certo. espero que sim.
> 
> professor guanabara? se for, estamos juntas o/

Que legal, eu também estou fazendo o curso com ele, estou apenas no início. Vou usar o Live Server também e ver se o que acontece. Muito Obrigada!
Desejo sucesso pra vcs!!! Muito bom... Mais mulheres em tecnologia 🖖

**Comment by tlopescavalcante** [comment]
cara, simplesmente não encontro o Live Server nas extensões pra baixar. Foi descontinuado também?

**Comment by Ireneu5** [comment]
alguém tem o link pra instalar a extensão no google?
(Aula do Prof. Guanabara)

**Comment by osvaldo53** [comment]
Também vim pela aula do guanabara :(

**Comment by MonicaDalosto** [comment]
Olá pessoal, 
Para utilizar o Live Server, não precisa baixar no PC. Basta adicionar como extensão do VSCode.
O que eu estou usando é o Live Server v.5.6.1:
![Screenshot 2022-01-08 at 10 26 52](https://user-images.githubusercontent.com/87082572/148639143-52cec146-9908-497f-bd21-888ebf819daa.png)
Bora codar! 🖖

**Comment by osvaldo53** [comment]
Consegui usar o Live Server, muito bom :)

**Comment by XBrunoDF** [comment]
> 

Aparentemente foi descontinuada!

**Comment by matheus-barth-correa** [comment]
> SNEEZRY

valeu cara

**Comment by slayerkk** [comment]
Estou usando o "Live Server".

**Comment by SilvioCruzDeveloper** [comment]
obrigado por endicarem o Live server

**Comment by cassiocpsgit** [comment]
Alguém indica outra opção para o navegador Edge?

**Comment by IsaEscaraber** [comment]
> SNEEZRY

O meu tambem não.

**Comment by Manharah23** [comment]
> Estou usando o Studio Visual Codes , a procura do plug-in Watch in Chrome , fui ao GitHub no fórum e nos comentários disseram q o LIVE SERTVE , fui instalei e ficou perfeito. Muito obrigado.

obrigado a dica funcionou

**Comment by Manharah23** [comment]
> Estou usando o "Live Server".

obrigado funciona no meu pc

**Comment by BrunoVsc09** [comment]
Caramba, valeu!
